### PR TITLE
In 148086495 add help message when allegations exist

### PR DESCRIPTION
**Pull request by saraOSI**

### CODE REVIEW

File: app/assets/javascripts/components/common/InfoMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const InfoMessage = ({message}) => (
+  <div className='row'>
+    <div className='col-md-12'>
+      <div className='alert-message info-message'>
```
Comment (jgedeon-osi) [inline app/assets/javascripts/components/common/InfoMessage.jsx]: I don't like calling the help message css styles alert-text, alert-message, etc. I'd prefer help-text, help-message, etc.

File: app/assets/stylesheets/alert.scss [hunk 2]
```diff
@@ -55,3 +55,31 @@
   padding-left: 2rem;
 }
 
+.alert-message {
```
Comment (jgedeon-osi) [inline app/assets/stylesheets/alert.scss]: I don't think these should be called alert. help or info might be more appropriate.

File: spec/javascripts/components/common/HelpMessageSpec.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import {shallow} from 'enzyme'
+import HelpMessage from 'components/common/HelpMessage'
+
+describe('HelpMessage', () => {
+  it('renders the message text passed to it', () => {
+    const messageText = 'Help me Obi-Wan Kenobi'
```
Comment (thomasramirez) [inline spec/javascripts/components/common/HelpMessageSpec.jsx]: Looking for the other half of the quote. It's kind of like shave and a haircut for toons...

File: spec/features/screening/allegations_cross_reports_spec.rb [hunk 4]
```diff
@@ -94,11 +96,72 @@
...
+      expect(page).to have_content('must be cross-reported to law enforcement')
+      find('label', text: /\ADistrict attorney\z/).click
+    end
+
+    within '#cross-report-card.edit' do
+      expect(page).to have_content('must be cross-reported to law enforcement')
+      find('label', text: /\ALaw enforcement\z/).click
+    end
+
+    within '#cross-report-card.edit' do
```
Comment (jgedeon-osi) [inline spec/features/screening/allegations_cross_reports_spec.rb]: is there a reason to have multiple within '#cross-report-card.edit' blocks? vs one block with the lines broken with blank lines, which is probably what I would prefer.

File: app/assets/javascripts/components/common/InfoMessage.jsx (hunk 1 above)
Comment (brunoOSI) [inline app/assets/javascripts/components/common/InfoMessage.jsx]: 👍

Comment (saraOSI) [inline app/assets/javascripts/components/common/InfoMessage.jsx]: There are additional changes coming that will add `error-message`, and both that class and `info-message` pull some shared styling from `alert-message`.

File: spec/features/screening/allegations_cross_reports_spec.rb (hunk 4 above)
Comment (saraOSI) [inline spec/features/screening/allegations_cross_reports_spec.rb]: I did this based on an improper assumption. All fixed now!

**Pull request merged**